### PR TITLE
feat(backing-image) Support download backing image

### PR DESCRIPTION
**Pull request by smallteeths**
image: wangsiye/longhorn-ui:399b94e

https://github.com/longhorn/longhorn/issues/3155

Signed-off-by: Siye Wang <siye@rancher.com>

### CODE REVIEW

File: src/services/backingImage.js [hunk 1]
```diff
@@ -28,6 +28,12 @@ export async function deleteBackingImage(params) {
   }
 }
 
+export async function download(params) {
+  if (params && params.name) {
+    window.location.href = `${ window.__pathname_prefix__ }${ window.__pathname_prefix__.endsWith('/') ? '' : '/'}v1/backingimages/${params.name}/download` // eslint-disable-line
```
Comment (innobead) [inline src/services/backingImage.js:33]: Will this open another target to download or replace the current window/tab?

**Review by innobead — COMMENTED** [review]
LGTM, just have one comment.

File: src/services/backingImage.js (hunk 1 above)
Comment (smallteeths) [reply src/services/backingImage.js:33]: If this is a download link, it will not open a new page, the browser will download the file directly. @innobead

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
Waiting for backend PRs merged first @shuo-wu

**Comment by innobead** [comment]
Merged first, because this should be independent of other functions, it's able to merge before the backend.

**Pull request merged**